config.go

```diff
@@ -480,6 +495,11 @@ func (t *Tracer) updateInstrumentationConfig(f func(cfg *instrumentationConfig))
 	}
 }
 
+// IgnoredTransactionURLMatchers returns the ignored transaction URLs
+func (t *Tracer) IgnoredTransactionURLMatchers() wildcard.Matchers {
+	return t.instrumentationConfig().ignoreTransactionURLs
+}
```

Comment:
On second thoughts, can you please make this method accept a URL and return a boolean? Package "wildcard" is internal, so we shouldn't be exposing it here. e.g.

```suggestion
// IgnoredTransactionURL reports whether transactions for the given URL should be ignored.
func (t *Tracer) IgnoredTransactionURLMatchers(url *url.URL) bool {
	return t.instrumentationConfig().ignoreTransactionURLs.MatchAny(url.String())
}
```